src/super_gradients/recipes/dataset_params/pascal_voc_segmentation.yaml

```diff
@@ -0,0 +1,55 @@
+img_size: 608
```

Comment:
Please rename the file to pascal_voc_segmentation_dataset_params.yaml.

Reply:
done